includes/classes/class-swsales-license.php

```diff
@@ -0,0 +1,136 @@
+				</div>
+			</div>
+            <div class="about-wrap">
+            <?php
+               if ( ! empty( $_REQUEST['sws-verify-submit'] ) && wp_verify_nonce( $_POST['sws-key-nonce'], 'sws-key-nonce' ) ) {
+                
+                $key = preg_replace("/[^a-zA-Z0-9]/", "", $_REQUEST['sws-license-key']);
+    
+                delete_option( 'sws_license_key' );
+    
+                $valid = sws_license_is_valid( $key, NULL, true );
+    
+                if ( $valid ) { ?>
+                    <div id="message" class="sitewide_sales_message sitewide_sales_success">
+                        <p><?php _e( 'Your license key has been validated.', 'sitewide-sales' ); ?></p>
+                    </div>
+                <?php } else {
+                    // handle error here.
+                }
+                update_option( 'sws_license_key', $key, 'no' );
+                }
+
+               $key = get_option( 'sws_license_key', '' );
+               $sws_license_check = get_option( 'sws_license_check', array( 'license' => false, 'enddate' => 0 ) );
+            ?>
+            <h2><?php esc_html_e( 'Sitewide Sales Support License', 'sitewide-sales' ); ?></h2>
+            
+
+            <div class="about-text">
+                <?php if(!sws_license_is_valid() && empty($key)) { ?>
+                    <p class="sitewide_sales_message sitewide_sales_alert"><strong><?php _e('Enter your support license key.</strong> Your license key can be found in your membership email receipt or in your <a href="https://www.paidmembershipspro.com/membership-account/?redirect_to=%2Fmembership-account%2F%3Futm_source%3Dplugin%26utm_medium%3Dpmpro-license%26utm_campaign%3Dmembership-account%26utm_content%3Dno-key" target="_blank">Membership Account</a>.', 'paid-memberships-pro' );?></p>
+                <?php } elseif(!sws_license_is_valid()) { ?>
+                    <p class="sitewide_sales_message sitewide_sales_error"><strong><?php _e('Your license is invalid or expired.', 'paid-memberships-pro' );?></strong> <?php _e('Visit the PMPro <a href="https://www.paidmembershipspro.com/membership-account/?redirect_to=%2Fmembership-account%2F%3Futm_source%3Dplugin%26utm_medium%3Dpmpro-license%26utm_campaign%3Dmembership-account%26utm_content%3Dkey-not-valid" target="_blank">Membership Account</a> page to confirm that your account is active and to find your license key.', 'paid-memberships-pro' );?></p>
+                <?php } else { ?>													
+                    <p class="sitewide_sales_message sitewide_sales_success"><?php printf(__('<strong>Thank you!</strong> A valid <strong>%s</strong> license key has been used to activate your support license on this site.', 'paid-memberships-pro' ), ucwords($sws_license_check['license']));?></p>
```

Comment:
Should be updated to use sitewide-sales text domain and link to Strangers Studios pages

Reply:
This has been updated.